ros/src/styx/server.py

```diff
@@ -50,26 +54,36 @@ def telemetry(sid, data):
         topic, data = msgs.popitem()
         sio.emit(topic, data=data, skip_sid=True)
 
+
 @sio.on('control')
 def control(sid, data):
     bridge.publish_controls(data)
 
+
 @sio.on('obstacle')
 def obstacle(sid, data):
-    bridge.publish_obstacles(data)
+    # Disable publishing obstacle data because these are publishing at high
```

Comment:
Interesting. Do you have any idea why the simulator publishes obstacle data if it's not intended to be used in the project? Perhaps the simulator designers thought an ambitious team might use that data in some non-predefined way?

Reply:
I think Udacity was planning to implement point clouds of obstacles later but never got to it and is backlogged with everyone dealing with just the traffic light.  Maybe future students will get to do something with it but for now I don't know why the simulator keeps spitting out junk data that gets published at a very high frequency.  For bandwidth and lag reduction, I want to disable it at least on the ROS side.

Reply:
Sounds good!